src/index.ts

```diff
@@ -149,9 +148,10 @@ export class DeviceMock extends EventTarget implements BluetoothDevice {
 
     private serviceMocks: { [service: string]: PrimaryServiceMock } = {};
 
-    constructor(public name: string, private services: BluetoothServiceUUID[]) {
+    constructor(public name: string, private services: BluetoothServiceUUID[], public id: string) {
```

Comment:
How about giving it a default value of `mock-device` or similar so the change is not breaking? It also makes the library easier to use when you need to mock just one device